.github/workflows/pr_reviews.yml

```diff
@@ -0,0 +1,46 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Adding Reviewers
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+jobs:
+  add_reviewers:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Add Reviewers
+      run: |
+        user_json=$(gh api users/${{ github.event.review.user.login }})
```

Comment:
we should probably explicitly specify the `accept` in the header to `-H "Accept: application/vnd.github+json" \` as per https://docs.github.com/en/rest/users/users?apiVersion=2022-11-28

Also maybe the version to `-H "X-GitHub-Api-Version: 2022-11-28" \`

Reply:
Thanks, I'll add these, good point.